Author: Apollon Oikonomopoulos <apoikos@debian.org>
Description: Do not build the Graphite RabbitMQ exporter
 We are missing the amqp client library. Since this is only optional for
 metrics-graphite to work, exclude GraphiteRabbitMQ from the build.
Last-Update: 2017-08-05
Forwarded: no
--- a/metrics-graphite/pom.xml
+++ b/metrics-graphite/pom.xml
@@ -34,4 +34,18 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.6.1</version>
+                <configuration>
+                    <excludes>
+                        <exclude>**/GraphiteRabbitMQ*</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
 </project>
